ads/google/a4a/test/test-traffic-experiments.js

```diff
@@ -76,26 +71,26 @@ describe('all-traffic-experiments-tests', () => {
 
     it('handles empty experiments list', () => {
       // Opt out of experiment.
-      sandbox.win.AMP_CONFIG['testExperimentId'] = 0.0;
+      toggleExperiment(sandbox.win, 'testExperimentId', false, true);
```

Comment:
If I read the code correctly, the randomness of `randomlySelectUnsetPageExperiments` is about "branches", which we still have tests covering that. On the other side, `AMP_CONFIG['testExperimentId'] = 0.0` is just turning off the experiment, which should have been done via toggleExperiment. Otherwise, it's just testing the logic inside toggleExperiment method. 

I could have mis-read the logic here, and that's why to have you to help :-).